src/client_capabilities.ts

```diff
@@ -26,58 +26,58 @@ export function prepareGetClientCapabilities(ua: string = '') {
     let signalUnknownCredential: boolean | undefined = false;
     let userVerifyingPlatformAuthenticator: boolean | undefined = false;
 
-    // If this is above macOS Safari 17.4 and below Safari 18.2, or above iOS
-    // 17.4 and below iOS 18.2, replace `conditionalMediation` with `conditionalGet`.
-    // To do so, copy the result and apply it to the polyfill.
-    const capabilities = version.safari174To182 || version.iOS174To182
+    // If the browser is above macOS Safari 17.4 and below Safari 18.2, or above
+    // iOS 17.4 and below iOS 18.2, replace `conditionalMediation` with
+    // `conditionalGet`.
+    if (version.safari174To182 || version.iOS174To182) {
```

Comment:
I'm understanding this logic to only try and call `getClientCapabilities()` on Safari or iOS. But `getClientCapabilities()` works in other browsers, too, which probably means it's supported on Android and Windows too. Maybe this becomes a check that `typeof PublicKeyCredential.getClientCapabilities === 'function'`, then below where you map `conditionalGet = capabilities?.conditionalMediation` could instead get wrapped in a check that `version.safari174To182 || version.iOS174To182`, otherwise it maps `conditionalGet = capabilities?.conditionalGet` 🤔

Reply:
If you go all the way down to line 109, the condition is `if (!PublicKeyCredential.getClientCapabilities || version.safari174To182 || version.iOS174To182) {` which applies the polyfill. This means when one of the following conditions meet:
- `PublicKeyCredential.getClientCapabilities` don't exist
- iOS 17.4 - 18.2
- macOS Safari 17.4 - 18.2

Then this line (32) has an else statement at line 48, which defines all the cases when `PublicKeyCredential.getClientCapabilities` does not exist.

Reply:
Alright, makes sense. Do you want to me to merge and deploy as-is, or do you want to address the test issues first?

Reply:
Could you address test issues?